EIPS/eip-x.md

```diff
@@ -25,61 +25,69 @@ The ERC-4626 Tokenized Vaults standard has helped to make yield-bearing tokens m
 
 ## Specification
 
 ### Definitions:
 
-The existing definitions from [ERC-4626](./eip-4626.mn) apply. In addition, this spec defines:
+The existing definitions from [ERC-4626](./eip-4626.md) apply. In addition, this spec defines:
 
-- request: a function call that initiates an asynchronous deposit/redemption flow
-- pending request: the state where a request has been made but is not yet claimable
-- claimable: the Vault's step of processing the request and enabling the user to claim corresponding shares (for async deposit) or assets (for async redeem)
-- claim: the corresponding Vault method to complete a request (e.g. `deposit` claims `shares` from `requestDeposit`)
-- operator: the account specified by the sender of the request which has has the right to claim a given request once it is claimable
-- asynchronous deposit Vault: a Vault that implements asynchronous requests for deposit flows
+- Request: a function call that initiates an asynchronous deposit/redemption flow
+- Pending: the state where a Request has been made but is not yet Claimable
+- Claimable: the state where a Request is processed by the Vault enabling the user to claim corresponding `shares` (for async deposit) or `assets` (for async redeem)
+- Claimed: the state where a Request is finalized by the user and the user receives the output token (e.g. `assets` for a deposit Request)
+- Claim function: the corresponding Vault method to bring a Request to Claimed state (e.g. `deposit` or `mint` claims `shares` from `requestDeposit`). Lower case claim always describes the verb action of calling a Claim function.
+- operator: the account specified by the sender of the Request which has has the right to claim a given Request once it is Claimable
+- asynchronous deposit Vault: a Vault that implements asynchronous Requests for deposit flows
 - asynchronous redemption Vault: a Vault that implements asynchronous redemption flows
-- fully asynchronous Vault: a vault that implements asynchronous requests for both deposit and redemption
+- fully asynchronous Vault: a Vault that implements asynchronous Requests for both deposit and redemption
 
 ### Request Flows
 
-EIP-X vaults MUST implement one or both of asynchronous deposit and redemption request flows. If either flow is not implemented in a request pattern, it MUST use the ERC-4626 standard synchronous interaction pattern. 
+EIP-X Vaults MUST implement one or both of asynchronous deposit and redemption Request flows. If either flow is not implemented in a Request pattern, it MUST use the ERC-4626 standard synchronous interaction pattern. 
 
-All EIP-X asynchronous tokenized vaults MUST implement ERC-4626, with the following overrides for request flows:
+All EIP-X asynchronous tokenized Vaults MUST implement ERC-4626, with the following overrides for Request flows:
```

Comment:
Maybe we should split up this section in two lists, so it's " with the following overrides for asynchronous deposit Vaults:" and separately "and with the following overrides for asynchronous redeem Vaults:"
```suggestion
All EIP-X asynchronous tokenized Vaults MUST implement ERC-4626, with the following overrides for Request flows:
```

Reply:
I thought about this too, I'll change I think it is clearer